modules/ROOT/pages/push/con_accessing-push-using-the-ui.adoc

```diff
@@ -0,0 +1,26 @@
+include::{partialsdir}/attributes.adoc[]
+
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: con_accessing-push-using-the-ui
+
+[id='{context}_con_accessing-push-using-the-ui']
+= Accessing {unifiedpush-service} using the User Interface
+
+The Administrative User Interface (AUI):
+
+* Displays all existing Push Applications.
+* Displays the _Variants_ tab of a Push Application
+** Details such as the number of sent messages and the amount of registered devices are display.
+* Displays details about the 3rd party Push Network.
+* Allows you to manage a registered device
+* Allows you to import and export device metadata
+* Send a Push Notification from (what?) code
```

Comment:
@jhellar , @pb82 - what does 'what?' stand for here? It originally stated "Send a Push Notification from code" - not sure what code this was referring to.

Reply:
@jstaffor @jhellar looks like a mistake in the original text. Maybe replace it with `Send a Push Notification through a REST Api`?